src/hooks/useWrapCallback.ts

```diff
@@ -38,18 +37,14 @@ export default function useWrapCallback(
     const sufficientBalance = inputAmount && balance && !balance.lessThan(inputAmount)
 
     if (inputCurrency === ETHER && currencyEquals(WETH[chainId], outputCurrency)) {
-      const { native, wrapped } = getChainCurrencySymbols()
```

Comment:
No longer getting the appropriate symbols for the network?

Reply:
Not in the original `src` now we do it in `custom`, it was a mistake to modify `src`